src/xdocs/config_misc.xml

```diff
@@ -326,19 +326,19 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
         <p>Example #4: Comment has distributed previous statement.</p>
         <source>
 1   public void foo11() {
-2       CheckUtils
-3           .getFirstNode(new DetailAST())
-4           .getFirstChild()
-5           .getNextSibling();
-6       // it is OK
+2     CheckUtils
+3     .getFirstNode(new DetailAST())
+4     .getFirstChild()
+5     .getNextSibling();
```

Comment:
These 3 lines should be indented. It was only the comment that was suppose to move.

Reply:
Done the change